README.md

```diff
@@ -114,36 +104,45 @@ In a nutshell, this means:
 
 
 ## Releasing a New Version (For Maintainers)
-Update version numbers and release using https://github.com/webpro/grunt-release-it:
+Prior to releasing, ensure that the CHANGELOG.md is updated to track any changes
+that have been made since the prior release.
+
+We increment version numbers and release using [release-it](https://github.com/webpro/release-it):
 
 ```bash
-vim CHANGELOG.md
-grunt release-it:<options>
-npm publish # access needed to publish new Addepar-owned npm packages
+npm install release-it -g # if not already installed
+release-it <options>
 ```
 
-By default, `grunt release-it` without options will increment the
-<patch> version number (X.Y.Z --> X.Y.(Z+1)) in the `VERSION` file,
-update the globals-based version of Ember Charts, and then commit
-the resulting changes to the ember-charts git repository.
+The local configuration file for `release-it` is named `.release.json`, found in the
+root directory of the repository.
+
+By default, `release-it` without options will increment the
+<patch> version number (`X.Y.Z` --> `X.Y.(Z+1)`) in the `VERSION` file and
+`package.json` file, and then commit the resulting changes to the ember-charts
+git repository.
 
 If you want to control the version number, use these options:
 
 ```bash
-grunt release-it:minor
-grunt release-it:major
-grunt release-it:X.Y.Z
+release-it minor # 1.2.3 -> 2.0.0
+release-it major # 1.2.3 -> 1.3.0
+release-it X.Y.Z # 1.2.3 -> X.Y.Z
```

Comment:
Crud, this is wrong. We've swapped `minor` and `major` version increments here.
`minor` changes should not increment the first part of the version number, while `major` changes should do so.